includes/api/class-give-api.php

```diff
@@ -150,8 +150,8 @@ public function __construct() {
 		add_filter( 'query_vars', array( $this, 'query_vars' ) );
 		add_action( 'show_user_profile', array( $this, 'user_key_field' ) );
 		add_action( 'edit_user_profile', array( $this, 'user_key_field' ) );
-		add_action( 'personal_options_update', array( $this, 'update_key' ) );
-		add_action( 'edit_user_profile_update', array( $this, 'update_key' ) );
+		add_action( 'personal_options_update', array( $this, 'generate_api_key' ) );
```

Comment:
@ravinderk I've not renamed the function `update_key`. I've just removed it as `generate_api_key` and `update_key` both are using similar code and functioning.